File: dongbinbook/previousTest/9-stringCompression.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string s) {
    int answer = 0;
    for (int i = 1; i <= s.length / 2; ++i) {
        int pos = 0;
        int cSize = s.length();

        while (1) {
            string cut = s.substr(pos, i);
            pos += i;

            if (pos >= s.length()) {
                break;
            }

            int count = 0;
            while (1) {
                if (cut.compare(s.substr(pos, i)) == 0) {
                    ++count;
                    pos += i;
                } else
                    break
            }
            if (count > 0) {
                len
            }
        }
    }
    return answer;
}

File: dongbinbook/previousTest/9-stringCompression.py
import math

s1 = "a a b b a c c c"  # length8 -> csize7
# 2a2ba3c
s2 = "ababcdcd ababcdcd"  # length16 -> csize 8
# 2ababcdcd
s3 = "abc abc ded e"  # length10 -> csize 8
# 3abcdede
s4 = "abcabc abcabc dedede dedede"  # length24 -> csize 14
# 2abcabc2dedede
s5 = "xababcdcdababcdcd"  # length17 -> csize 17(x)


def solution(s):
    answer = len(s)
    tryDiv = len(s) // 2
    

solution(s1)